src/coreclr/jit/typeinfo.cpp

```diff
@@ -20,32 +20,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 bool Compiler::tiCompatibleWith(const typeInfo& child, const typeInfo& parent, bool normalisedForStack) const
 {
-#ifdef DEBUG
-#if VERBOSE_VERIFY
-    if (VERBOSE && tiVerificationNeeded)
-    {
-        printf("\n");
-        printf(TI_DUMP_PADDING);
-        printf("Verifying compatibility against types: ");
-        child.Dump();
-        printf(" and ");
-        parent.Dump();
-    }
-#endif // VERBOSE_VERIFY
-#endif // DEBUG
-
-    bool compatible = typeInfo::tiCompatibleWith(info.compCompHnd, child, parent, normalisedForStack);
-
-#ifdef DEBUG
-#if VERBOSE_VERIFY
-    if (VERBOSE && tiVerificationNeeded)
-    {
-        printf(compatible ? " [YES]" : " [NO]");
-    }
-#endif // VERBOSE_VERIFY
-#endif // DEBUG
-
-    return compatible;
+    return typeInfo::tiCompatibleWith(info.compCompHnd, child, parent, normalisedForStack);
```

Comment:
Does anything call `Compiler::tiCompatibleWith` this now? 

If not, there is more cleanup we can do.

Reply:
I'll check and see.

Reply:
There are several places calling `Compiler::tiCompatibleWith`.